packages/flutter/test/material/dropdown_test.dart

```diff
@@ -2223,7 +2223,7 @@ void main() {
     final Element element = tester.element(find.byKey(const ValueKey<String>('two')).last);
     final FocusNode node = Focus.of(element);
     expect(node.hasFocus, isTrue);
-  }, skip: kIsWeb);
+  }, skip: isBrowser); // https://github.com/flutter/flutter/issues/55320
```

Comment:
Just for my own reference, when should we use `isBrowser` vs `kIsWeb`?

Reply:
It looks like they boil down to the same thing `identical(0, 0.0)`, just from different places. I wonder if we should simplify to just one..
I've most often seen `isBrowser`, so I've been updating it for uniformity.

Reply:
I found another place where it's defined, but it's private for `packages/flutter/lib/src/services/text_input.dart` (I made a [PR](https://github.com/flutter/flutter/pull/55333) to see if we can remove it). 

If we look at where `isBrowser` is defined (`packages/flutter_test/lib/src/platform.dart`), it seems to have wrappers where it first checks if `identical(0, 0.0)` returns `true` or `false` before checking if the platform is Windows, MacOS and Linux. I wonder if that's something that check is unique to `flutter_test` and is more comprehensive in some way?

Reply:
Found the PR explaining this: https://github.com/flutter/flutter/pull/33780 -- I think it's to avoid conflating skipping when running on a platform (`Platform.isWindows`) vs skipping when running on a browser on that platform (`isWindows`)